stdlib/3/multiprocessing/pool.pyi

```diff
@@ -43,11 +46,11 @@ class Pool(ContextManager[Pool]):
     def imap(self,
              func: Callable[..., Any],
              iterable: Iterable[Any] = ...,
-             chunksize: Optional[int] = ...) -> Iterable[Any]: ...
+             chunksize: Optional[int] = ...) -> IMapIterator: ...
     def imap_unordered(self,
                        func: Callable[..., Any],
                        iterable: Iterable[Any] = ...,
-                       chunksize: Optional[int] = ...) -> Iterable[Any]: ...
+                       chunksize: Optional[int] = ...) -> IMapIterator: ...
```

Comment:
This technically returns an IMapUnorderedIterator (a subclass of IMapIterator), but there doesn't seem to be any interface difference with IMapIterator, so I think we can safely ignore that.

Reply:
Thanks for checking, my opinion is the same.